Jwt protect endpoints (#889)
change-metrics/monocle: README.md

## Configuration

The Monocle configuration is splitted between a configuration file and various environment
variables.

[Environment variables](#environment-variables) configure system settings and secrets. Any changes require
a restart of the Monocle API. 

[The configuration file](#configuration-file) contains non sensitive data and could be exposed in a Git repository.
A CI/CD flow could be enabled on top of the Git repository to allow Monocle' users to

> propose configuration changes. The Monocle API tracks the configuration file's modification date and reloads transparently in case of changes.  

Reviewer: ```suggestion
propose configuration changes. The Monocle API automatically reload the configuration when it changes.
```

### Environment variables

For a local deployment, default settings are fine.

The following settings are available in the `.env` file:

- `MONOCLE_PUBLIC_URL=<url>` to configure the public URL to access the UI and API.

>   This is required for the authentication provider redirection.

Reviewer: ```suggestion
  The URL is required for the user redirection from the authentication provider.
```
